v3iofs/fs.py

```diff
@@ -211,39 +214,58 @@ def container_info(container):
     }
 
 
-_missing = object()
-_extra_obj_attrs = [
-    # dest, src, convert
-    ('created', 'creating_time', parse_time),
-    ('atime', 'access_time', parse_time),
-    ('mode', 'mode', lambda v: int(v[1:], base=8)),  # '040755'
-    ('gid', 'gid', lambda v: int(v, 16)),
-    ('uid', 'uid', lambda v: int(v, 16)),
-]
+def prefix_path(container_name, prefix):
+    if not isinstance(prefix, str):
+        prefix = prefix.prefix
+    # prefix may already have a leading / - check and add
+    # if not present
+    if not prefix.startswith("/"):
+        prefix = f'/{prefix}'
+    return unslash(f'/{container_name}{prefix}')
+
+
+def prefix_info(container_name, prefix):
+    return info_of(container_name, prefix, 'prefix')
+
 
+def object_path(container_name, object):
+    # object.key already have a leading /
+    return f'/{container_name}{object.key}'
```

Comment:
I think this is very confusing, in previous commit I made sure `object` does *not* have a leading `/'.
If this is an issue, I prefer to have `f'{container_name}/{object.key}' and then normalize it.